app/presenters/concerns/assessment_tasks/assessment_detail_presenter.rb

```diff
@@ -29,31 +29,31 @@ def assessment_details_link
     end
 
     def assessment_details_link_url
-      case status.humanize
+      case status
 
-      when "Not started"
+      when "not_started"
         new_planning_application_assessment_detail_path(planning_application, category: category)
-      when "In progress"
+      when "in_progress"
         edit_planning_application_assessment_detail_path(planning_application, assessment_detail)
-      when "Completed"
+      when "completed"
         planning_application_assessment_detail_path(planning_application, assessment_detail)
       else
-        raise ArgumentError, "The status provided: '#{status}' is not valid"
+        raise ArgumentError, "The status provided: '#{status.humanize}' is not valid"
       end
     end
 
     def assessment_details_status_tag
       classes = ["#{govuk_tag_class} app-task-list__task-tag"]
 
       tag.strong class: classes do
-        status
+        assessment_details_status
       end
     end
 
     def assessment_details_status
-      return "Not started" unless assessment_detail
+      return I18n.t("assessment_details.not_started") unless assessment_detail
 
-      assessment_detail.status.humanize
+      I18n.t("assessment_details.#{status}")
```

Comment:
Alternatively we could change the status enum in the `AssessmentDetail` model to be "complete" instead of "completed". But I don't wand to get into a habit of making logic or data changes every time there's a change in the front end copy! Using translations seems simpler, because if tomorrow we decide that the tag should read "Done", "Finished" or "Banana" we only have to change it in one place!